src/site/site.xml

```diff
@@ -56,7 +57,6 @@
       <item name="1.8" href="javadocs/api-1.8/index.html"/>
```

Comment:
Instead of keep this list, why not point to javadoc.io like we do here https://commons.apache.org/proper/commons-io/ ?

Reply:
> Instead of keeping this list, why not point to javadoc.io like we do here https://commons.apache.org/proper/commons-io/ ?

Sounds like a good idea, but possibly a bit off-topic for specific changes as these ones: according to I've seen in _commons-io_, it seems that the (commons-compress) download page would also need some work in order to list previous versions, together with the javadoc links. Could I suggest creating a follow-up on this? 😉

Reply:
> Could I suggest creating a follow-up on this? 😉

I've created the follow-up: [COMPRESS-593](https://issues.apache.org/jira/projects/COMPRESS/issues/COMPRESS-593)
Please add any relevant suggestions there. 🙂